src/components/ApplicationGroup/Details/EnvCDDetails/EnvCDDetails.tsx

```diff
@@ -131,6 +132,30 @@ export default function EnvCDDetails({ filteredAppIds }: AppGroupDetailDefaultTy
             setHasMore(true)
             setHasMoreLoading(true)
         }
+        let _triggerId = deploymentHistoryResult.result?.cdWorkflows?.[0]?.id
```

Comment:
make a util to get triggerId instead of let

Reply:
What difference will it make?